tests/foreman/ui/test_host.py

```diff
@@ -1313,7 +1313,7 @@ def test_global_registration_form_populate(
 def test_global_registration_with_capsule_host(
     session,
     capsule_configured,
-    rhel7_contenthost,
```

Comment:
We're covering all of the host registrations in tests/foreman/api/test_registration.py::test_host_registration_end_to_end
With UI there's no significant change in the registration process, so there's not much we would get if we cover all versions, and I think covering only rhel7 or rhel8 contenthost makes sense

Reply:
Fair point, there is also UI e2e test for registration at tests/foreman/ui/test_registration.py::test_positive_global_registration_end_to_end
that runs all the supported RHELs.

Given that, instead of using more RHEL versions here, I would at least update `rhel7_contenthost` to `rhel8_contenthost` as RHEL 8 is IMO more common and recommended nowadays.

Reply:
yes sure, it'll make sense

Reply:
Done, moved to RHEL 8.